src/config.rs

```diff
@@ -99,6 +99,11 @@ pub struct Discv5Config {
     /// A custom executor which can spawn the discv5 tasks. This must be a tokio runtime, with
     /// timing support. By default, the executor that created the discv5 struct will be used.
     pub executor: Option<Box<dyn Executor + Send + Sync>>,
+
+    /// The max limit for peers with unreachable ENRs. Benevolent examples of such peers are peers
+    /// that are discovering their externally reachable socket and peers behind symmetric NAT.
+    /// Default is no limit. Minimum is 1.
+    pub unreachable_enr_limit: Option<usize>,
```

Comment:
ip voting is the reason even the lightest of clients must accepts at least one session with a peer with an unreachable enr. rn now sessions with unreachable enr peers are unlimited, suggestion on a limit value?

Reply:
Really? That's low. That will really make node operators port forward to get out of their symmetric NAT situation...and make nodes upgrade to discv5.2 so that they aren't the only ones left serving an unlimited number of symmetrically NAT:ed nodes...which is good for the network in the long run, but in the short run?